doto-backend/swagger.json

```diff
@@ -0,0 +1,271 @@
+    }
+  ],
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+   
+    "/schedule" : {
+      "post" : {
+        "tags": [
+          "Task"
+        ],
+        "description": "Create new task in system",
+        "parameters": [
+          {
+            
+              
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/Task"
+            }
+
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "returns" : [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "New Task is created"
+          }
+        }
+
+      },
+      "put" :{
+        "tags": [
+          "Task"
+        ],
+        "description": "Edit Task in System",
+        "parameters": [
+          {
+            
+            
+            "in": "body",
+            
+            
+            "schema": {
+              "$ref": "#/definitions/Task"
+            }
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "returns" : [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Task Details Edited"
+          }
+        }
+
+
+      },
+      "get" :{
+        "tags": [
+          "Task"
+        ],
+        "description": "Get all Tasks for user",
+        "parameters": [
+          {
+            
+            "in": "User",
+            "description" : " The user (String)"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "returns" : [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Here are all the Tasks",
+            "schema": {
+              "$ref": "#/definitions/Tasks"
+            }
+          }
+        }
+
+      },
+      "delete":{
+        "tags": [
+          "Task"
+        ],
+        "description": "Delete Task in System",
+        "parameters": [
+        
+          {
+            "in" : "taskId",
+            "description" : "The task to delete"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "returns" : [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Task Deleted"
+          }
+        }
+
+      }
+    },
+
+    "/login" :{
+      "post" : {
+
+      }
+    },
+
+    "/register" : {
+      "post" :{
+        "tags": [
+          "Task"
+        ],
+        "description": "Create new User in system",
+        "parameters": [
+          {
+            
+              
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/User"
+            }
+
+          }
+        ],
+        "produces": [
+          "application/json"
```

Comment:
Are produces and returns the same thing?

Reply:
tbh, not sure